### PR TITLE
feat: add support for forecasting metadata types

### PR DESCRIPTION
**Pull request by jdkgabri**
## Explain your changes

Added forecasting related metadata types.
---

## Does this close any currently open issues?

---

closes #

- [ ] Jest test added to check the fix is applied.

## Any particular element that can be tested locally

---

## Any other comments

Tested on Windows 11 with API version 57.

Not incrementing API version/release. To be decided up to the mantainers.

---

### CODE REVIEW

File: src/metadata/v57.json [hunk 1]
```diff
@@ -1,4 +1,39 @@
 [
+  {
+    "directoryName": "forecastingFilters",
+    "inFolder": false,
+    "metaFile": false,
+    "suffix": "forecastingFilter",
+    "xmlName": "ForecastingFilter"
+  },
+  {
+    "directoryName": "forecastingFilterConditions",
```
Comment (jdkgabri) [inline src/metadata/v57.json:10]: Documentation states that folder and suffix start with Uppercase but it is wrong: https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_forecastingfiltercondition.htm

Proof:

image

File: src/metadata/v57.json [hunk 2]
```diff
@@ -1,4 +1,39 @@
 [
+  {
```
Comment (scolladon) [inline src/metadata/v57.json:2]: **remarks:** those metadata are available since v55

They should also be added in v55 and v56

**Review by scolladon — CHANGES_REQUESTED** [review]
Thank you very much, nice job so far

I have one comment, please address it so we can include this contribution

File: src/metadata/v57.json (hunk 2 above)
Comment (jdkgabri) [reply src/metadata/v57.json:2]: All right. On it.

**Review by scolladon — APPROVED** [review]
LGTM

thanks for this contribution

**Pull request merged**